man/systemd-notify.xml

```diff
@@ -191,6 +191,24 @@
         escaped as <literal>\;</literal>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--fd=</option></term>
+
+        <listitem><para>Send a file descriptor along with the notification message. This is useful when
+        invoked in services that have the <varname>FileDescriptorStoreMax=</varname> setting enabled, see
+        <citerefentry><refentrytitle>systemd.service</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+        for details. The specified file descriptor must be passed to <command>systemd-notify</command> when
```

Comment:
Maybe mention how this can be done in bash? https://github.com/containers/bubblewrap/blob/main/demos/bubblewrap-shell.sh is a good example

Reply:
yeah, this is obscure enough to warrant a specific mention and example here